app/services/facets.rb

```diff
@@ -0,0 +1,11 @@
+module Facets
```

Comment:
Can we pass classes instead for most of these? There is no need to pass the symbols for most:

```

Facets.configuration.register(PuppetFacet) do 
  
end
```

.register could take a class, and a symbol only for those cases where the class name would be too long or undesirable to convert into a method. For instance, Katello::Subscriptions could be called as `.register(Katello::SubscriptionFacet, :subscription_facet)` rather than `.katello_subscription_facet`?